test/application/respond.js

```diff
@@ -374,7 +374,7 @@ describe('app.respond', () => {
     describe('with custom status=700', () => {
       it('should respond with the associated status message', async() => {
         const app = new Koa();
-        statuses['700'] = 'custom status';
```

Comment:
this is a breaking change for users who want to customize status code

Reply:
IMO we need to provide a more controllable way to customize status code.

Reply:
> this is a breaking change for users who want to customize status code

According to what I understand form you, we should leave this PR to Koa@3  !!

Reply:
> IMO we need to provide a more controllable way to customize status code.

Perhaps we can provide a method to store, set, cache.. custom messages for custom codes.

I think this approach is simple and clean.